libvast/src/system/accountant.cpp

```diff
@@ -81,6 +99,29 @@ void record(accountant_actor* self, const std::string& key, T x,
     st.flush_pending = true;
     self->delayed_send(self, 10s, flush_atom::value);
   }
+#else
+  auto& st = self->state;
+  auto& sys = self->system();
+  auto actor_id = self->current_sender()->id();
+  auto node = self->current_sender()->node();
+  if (!st.builder) {
+    auto layout
+      = record_type{{"ts", time_type{}},    {"nodeid", string_type{}},
+                    {"aid", count_type{}},  {"actor_name", string_type{}},
+                    {"key", string_type{}}, {"value", string_type{}}}
+          .name("vast.account");
+
+    auto slice_type = get_or(sys.config(), "system.table-slice-type",
+                             defaults::system::table_slice_type);
+    st.builder = factory<table_slice_builder>::make(slice_type, layout);
+    VAST_DEBUG(self, "obtained builder");
+  }
+  if (!st.builder->add(ts, to_string(node), actor_id, st.actor_map[actor_id],
+                       key, to_string(x)))
```

Comment:
Can this actually fail? If not (because the programmer dictates the schema), then we should probably make this an assertion.

Reply:
Fixed